packages/salesforcedx-vscode-core/src/conflict/messages.ts

```diff
@@ -33,6 +35,36 @@ export const getConflictMessagesFor = (logName: string): ConflictDetectionMessag
             .build()
             .toString()
       }
+    ],
+    [
+      'retrieve_with_sourcepath',
+      {
+        warningMessageKey: retrieveWarningMessageKey,
+        commandHint: inputs =>
+          (Array.isArray(inputs) ? inputs : [inputs])
+            .map(input =>
+              new SfCommandBuilder()
+                .withArg('project:retrieve:start')
+                .withFlag('--sourcepath', input)
+                .build()
+                .toString()
+            )
+            .join('\n  ')
+      }
+    ],
+    [
+      'project_retrieve_start_default_scratch_org',
+      {
+        warningMessageKey: retrieveWarningMessageKey,
+        commandHint: () => nls.localize('pull_conflicts_error')
+      }
+    ],
+    [
+      'project_deploy_start_default_scratch_org',
+      {
+        warningMessageKey,
+        commandHint: () => nls.localize('push_conflicts_error')
```

Comment:
I'm trying to follow all the layers of abstraction here.  What I *think* i see is that only `retrieve` is going to do anything with the "commandHint" but we're creating them on all the other message types?